fitlins/interfaces/nistats.py

```diff
@@ -141,6 +141,10 @@ def dscalar_from_cifti(img, data, name):
 
 
 class FirstLevelModel(NistatsBaseInterface, FirstLevelEstimatorInterface, SimpleInterface):
+    def __init__(self, errorts=False, *args, **kwargs):
+        super(FirstLevelModel, self).__init__(*args, **kwargs)
+        self.errorts = errorts
```

Comment:
```suggestion
    def __init__(self, *args, **kwargs):
        # Do not error on errorts being passed, but don't try to use it
        # This provides a common API with AFNI's FirstLevelModel
        kwargs.pop("errorts", None)
        super(FirstLevelModel, self).__init__(*args, **kwargs)
```

Reply:
I'm curious about this. It's a nice workaround. Is this a typical pattern or just an idiosyncratic hack until a common API is arrived upon?  Should a logging warning be emitted here?

Reply:
I consider it a hack. I don't think a warning is a good idea because it's nothing the user can do anything about, so it would just be a note to self, which this comment also serves as. We could add an `XXX` to make it a little more findable if we consider it worth prioritizing.

Reply:
Good to know. Thanks. I'm happy with it as is.